tests/roles/dataplane_adoption/tasks/main.yaml

```diff
@@ -140,7 +140,7 @@
           deployStrategy: {{ deployStrategy }}
           hostName: standalone
           node:
-            ansibleVars: |
+            ansibleVars:
```

Comment:
so does it mean we can expect the currently running job in the periodic line to fail until we merge this now?

right now the latest run in history is from yesterday  https://review.rdoproject.org/zuul/builds?job_name=periodic-centos-9-standalone-to-crc-data-plane-adoption-antelope

but currently in the queue https://review.rdoproject.org/zuul/status

periodic-centos-9-standalone-to-crc-data-plane-adoption-antelope waiting